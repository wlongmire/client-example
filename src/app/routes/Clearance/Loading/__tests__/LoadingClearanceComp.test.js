/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Loading } from '../index'

jest.mock('./../../../../actions/userActions', () => 'testActions')
jest.mock('./../../../../actions/submissionActions', () => 'testActions')

const initialState = {
  result: {},
  input: {
    insuredAddress: {
      primaryInsuredAddress: 'Test 999',
      primaryInsuredState: 'New York',
      primaryInsuredCity: 'Test 999',
      primaryInsuredZipcode: 'Test 999',
    },
    primaryInsuredName: 'Test 999',
    projectAddress: {
      projectAddress: 'Test 999',
      projectCity: 'Test 999',
      projectState: 'California',
      projectZipcode: 'Test 999'
    }
  },
  user: {
    broker: 'test-broker-id'
  }

}

// Snapshot for Home Header Component
describe('>>> Clearance Result Component--- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Loading
        input={initialState.input}
        handleSubmit={jest.fn()}
        user={initialState.user}
        handleCancel={jest.fn()}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})
