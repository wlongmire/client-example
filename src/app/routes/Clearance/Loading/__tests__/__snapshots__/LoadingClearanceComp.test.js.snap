// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Clearance Result Component--- Snapshot +++capturing Snapshot with 1`] = `
<form>
  <h3>
    Checking Prior Submissions
  </h3>
  <h4>
    Please Wait while we scan for matches.
  </h4>
  <div
    className="loadingImg"
  >
    <img
      src="undefined/images/ajax-loader.gif"
    />
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      className="btn"
      disabled={false}
      onClick={[Function]}
    >
      Return to Clearance Form
    </Button>
  </ButtonGroup>
</form>
`;
